appveyor.yml

```diff
@@ -0,0 +1,17 @@
+environment:
+  matrix:
+  - TARGET: x86_64-pc-windows-msvc
+  - TARGET: i686-pc-windows-msvc
+  - TARGET: i686-pc-windows-gnu
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
```

Comment:
Is this testing nightly? 
I think we should only test stable on Windows, because nightly can fail and we're testing other channels on linux.
See [appveyor example](https://github.com/seanmonstar/reqwest/blob/master/.appveyor.yml).